app/src/main/kotlin/com/mapzen/erasermap/controller/MainActivity.kt

```diff
@@ -587,6 +585,10 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
     }
 
     override fun showReverseGeocodeFeature(features: List<Feature>) {
+        hideSearchResults()
+        centerOnCurrentFeature(features)
```

Comment:
I think we shouldn't move the map automatically in this case. The user will have their finger pressed on one location and if the map moves while their finger is still on the screen then they will no longer be pressing in the same spot.